pkg/bpf/perf.go

```diff
@@ -138,6 +138,35 @@ import (
 const (
 	EventsMapName   = "cilium_events"
 	MAX_POLL_EVENTS = 32
+
+	PERF_TYPE_HARDWARE   = 0
+	PERF_TYPE_SOFTWARE   = 1
+	PERF_TYPE_TRACEPOINT = 2
+	PERF_TYPE_HW_CACHE   = 3
+	PERF_TYPE_RAW        = 4
+	PERF_TYPE_BREAKPOINT = 5
+
+	PERF_SAMPLE_IP           = 1 << 0
```

Comment:
Don't you prefer `PERF_SAMPLE_IP   = 1 << iota` and the remaining ones are automatically assigned?